daemon/cmd/hubble.go

```diff
@@ -267,3 +268,9 @@ func (d *Daemon) LookupSecIDByIP(ip net.IP) (id ipcache.Identity, ok bool) {
 	}
 	return id, false
 }
+
+// GetK8sStore returns the k8s watcher cache store for the given resource name.
+// It implements hubble parser's StoreGetter.GetK8sStore
+func (d *Daemon) GetK8sStore(name string) k8scache.Store {
+	return d.k8sWatcher.GetStore(name)
+}
```

Comment:
Please add a warning somewhere that the objects returned by these stores can't be used to create / update objects into k8s as well as these structures should only be used for reading.

Reply:
Yes, that's true. Updated the comments. Thanks!